src/Nordea/Components/Table.elm

```diff
@@ -0,0 +1,111 @@
+module Nordea.Components.Table exposing
+    ( tbody
+    , td
+    , th
+    , thead
+    , tr
+    , view
+    )
+
+import Css
+    exposing
+        ( alignSelf
+        , backgroundColor
+        , borderBottom3
+        , center
+        , column
+        , displayFlex
+        , flexDirection
+        , height
+        , hover
+        , nthChild
+        , padding
+        , padding4
+        , px
+        , rem
+        , row
+        , solid
+        )
+import Css.Global exposing (descendants, typeSelector)
+import Html.Styled as Html exposing (Attribute, Html)
+import Html.Styled.Attributes exposing (css)
+import Nordea.Components.Text as Text
+import Nordea.Resources.Colors as Colors
+
+
+view : List (Attribute msg) -> List (Html msg) -> Html msg
+view attrs children =
+    Html.table
+        (css [ displayFlex, flexDirection column ]
+            :: attrs
+        )
+        children
+
+
+{-| Groups the header content in a table
+-}
+thead : List (Attribute msg) -> List (Html msg) -> Html msg
+thead attrs children =
+    Html.thead
+        (css
+            [ height (rem 3.0)
+            , borderBottom3 (px 1) solid Colors.grayLight
+            ]
+            :: attrs
+        )
+        children
+
+
+{-| Defines a row in a table
+-}
+tr : List (Attribute msg) -> List (Html msg) -> Html msg
+tr attrs children =
+    Html.tr
+        (css
+            [ displayFlex
+            , flexDirection row
+            ]
+            :: attrs
+        )
+        children
+
+
+{-| Defines a header cell in a table
+-}
+th : List (Attribute msg) -> List (Html msg) -> Html msg
+th attrs children =
+    Html.th
+        (css [ padding4 (rem 1.0) (rem 0.75) (rem 0.75) (rem 0.75) ] :: attrs)
+        [ Text.init Text.TextSmallHeavy
+            |> Text.view [] children
+        ]
+
+
+{-| Groups the body content in a table
+-}
+tbody : List (Attribute msg) -> List (Html msg) -> Html msg
+tbody attrs children =
+    Html.tbody
+        (css
+            [ displayFlex
+            , flexDirection column
+            , descendants
+                [ typeSelector "tr"
+                    [ height (rem 4.5)
+                    , hover [ backgroundColor Colors.blueCloud |> Css.important ]
```

Comment:
We must remember to use theme colors for most blue colors 🙂 Also, is the Css.important needed?


```suggestion
                    , hover [ Themes.backgroundColor Themes.SecondaryColor Colors.blueCloud ]
```

Reply:
Aah thanks for the reminder 😍
Yeah, since we're coloring every other row, these already have a background color, so to override this I thiiink the `Css.important` is necessary

Reply:
Just a guess, but maybe changing `descendants` to `children` gives this more specificity and enough to override the alternate color without important

Reply:
Aaah, I never really checked out the difference between the two. I'll try 🙌

Reply:
Tried changing to `children`, looks like `Css.important` is still needed 👎